wire-grpc-server-generator/src/test/java/com/squareup/wire/kotlin/grpcserver/FileDescriptorGeneratorTest.kt

```diff
@@ -46,7 +46,9 @@ internal class FileDescriptorGeneratorTest {
       |package test;
       |import "imported.proto";
       |
-      |message Test {}
+      |message Test {
+      |  optional Imported imported = 1;
```

Comment:
A side effect of this change is that `descriptor.dependencies.size` was 0 without this change becuase the unused import is getting pruned away. I'm not sure how much of an issue that is in reality though?

Reply:
no problem I think